appside/frames/likert.js

```diff
@@ -0,0 +1,114 @@
+"use strict";
+
+/**
+ * Rendering (View) code for likert frame
+ * @author Rachel Sitt
+ */
+
+
+/**
+ * A frame composed of a list of things.
+ * 
+ * Things in the list are to be rendered as a list of checkboxes.
+ **/
+class LikertFrame extends Frame {
+
+    /**
+     * Construct LikertFrame from an object
+     * 
+     * @param frame_data -- Object containing the frame's data. Expected fields:
```

Comment:
docs need to be updated
- specify the individual fields of frame_data that are expected.

Reply:
updated